.github/workflows/bazelBuildAndTest.yml

```diff
@@ -0,0 +1,30 @@
+name: Bazel Build and Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      release_id:
```

Comment:
I don't think you need this release_id thing.

Reply:
You are right we don't want to push  artifacts here, will remove this. thansk!